esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -6100,14 +6081,19 @@ bool CWsDfuEx::onDFUFileAccess(IEspContext &context, IEspDFUFileAccessRequest &r
 {
     try
     {
-        SecAccessFlags accessType = translateToSecAccessFlags(req.getAccessType());
+        IConstDFUFileAccessRequestBase &requests = req.getRequestBase();
```

Comment:
requests (plural) is a bit confusing, change to requestBase ?

Reply:
ok